### DISCUSSION TITLE
Can you ran this server standalone or should you proxy it?

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/WatsonWebserver#75] [Q&A]
Hi,

first of all also a very cool project. I am interessted to replace my work with asp.net with this. I am currently conncerned how safe it is to use, like does it have dos prevention or any security features? Or is it better just to proxy it trough e.g. apache2?

Thanks!

**Comment by jchristn** [comment]
Hi @flo2000ace I use it regularly in projects that are exposed to the Internet alongside a network firewall and operating system firewall.  It has some basic security capabilities like IP filtering.  For DoS prevention you would need an external solution.  Cheers
